### ISSUE TITLE
Plotting issue in finalize_hrtf_simulation.py

### ISSUE THREAD

**Issue by TheTrackdriver** [Any2HRTF/Mesh2HRTF#149]
Everything has worked out fine except for the plotting process. I don't get any .pngs in the project merged folder. I have copied the output from the CMD window below. I did run the "pip list --outdated" and got a few hits which I uppdated. That "numpy" thing was already up to date btw. Any suggestions?

--------------------------------------------------------------
C:\Users\xxxx\anaconda3\envs\mesh2hrtf\lib\site-packages\pyfar\classes\coordinates.py:2169: PyfarDeprecationWarning: 'This function will be deprecated in pyfar 0.8.0. Use properties and slicing instead.'
  warnings.warn((
C:\Users\xxxx\anaconda3\envs\mesh2hrtf\lib\site-packages\pyfar\classes\coordinates.py:792: PyfarDeprecationWarning: 'This function will be deprecated in pyfar 0.8.0 in favor of the `spherical_*` properties.'
  warnings.warn((
C:\mesh2hrtf-tools\Finalize_HRTF_simulation\finalize_hrtf_simulation.py:296: MatplotlibDeprecationWarning: The get_cmap function was deprecated in Matplotlib 3.7 and will be removed in 3.11. Use ``matplotlib.colormaps[name]`` or ``matplotlib.colormaps.get_cmap()`` or ``pyplot.get_cmap()`` instead.
  cmap=mpl.cm.get_cmap(name='hot_r'),

  WARNING: plotting command crashed. Likely reasons are:
              1 - obsolete python packages (especially numpy) - try running: "pip list --outdated"
                  (to update a package use for example: "pip install --upgrade numpy"
              2 - actual bug in this script (debug, or open an issue if you need this plot)

Plotting diagnostic plots ...

-------READY--------------------------------------------------
Hit Enter to close - Plotting is complete   :)

**Comment by f-brinkmann** [comment]
Hi, and thanks for reporting! Can you let me know which mesh2hrtf version you are using? You can find the version number here: https://github.com/Any2HRTF/Mesh2HRTF/blob/master/VERSION. If you are not using the latest version alread,  please try if the problem persists, if you update mesh2hrtf.

**Comment by TheTrackdriver** [comment]
Hi, it was version 1.2.0. I updated to 1.2.1 but still no luck.

--------------------------------------------------------------
C:\Users\xxxx\anaconda3\envs\mesh2hrtf\lib\site-packages\pyfar\classes\coordinates.py:2169: PyfarDeprecationWarning: 'This function will be deprecated in pyfar 0.8.0. Use properties and slicing instead.'
  warnings.warn((
C:\Users\xxxx\anaconda3\envs\mesh2hrtf\lib\site-packages\pyfar\classes\coordinates.py:792: PyfarDeprecationWarning: 'This function will be deprecated in pyfar 0.8.0 in favor of the `spherical_*` properties.'
  warnings.warn((
C:\mesh2hrtf-tools\Finalize_HRTF_simulation\finalize_hrtf_simulation.py:296: MatplotlibDeprecationWarning: The get_cmap function was deprecated in Matplotlib 3.7 and will be removed in 3.11. Use ``matplotlib.colormaps[name]`` or ``matplotlib.colormaps.get_cmap()`` or ``pyplot.get_cmap()`` instead.
  cmap=mpl.cm.get_cmap(name='hot_r'),

  WARNING: plotting command crashed. Likely reasons are:
              1 - obsolete python packages (especially numpy) - try running: "pip list --outdated"
                  (to update a package use for example: "pip install --upgrade numpy"
              2 - actual bug in this script (debug, or open an issue if you need this plot)

Plotting diagnostic plots ...

-------READY--------------------------------------------------
Hit Enter to close - Plotting is complete   :)

**Comment by f-brinkmann** [comment]
Ok, I'll try to reproduce. Can you let me know which command fails and export your environment so I can see the versions of the packages you are using? In any case, the deprecations indicate that something should be updated on our side.

**Comment by jdk149** [comment]
hallo guys! I have the same problem wit the plotting command. the specific situation is when I launching [finalize_hrtf_simulation.py] for the two folders (myHRTF_project_L&myHRTF_project_R), the other command just work well but Run console says just like the [TheTrackdriver](https://github.com/TheTrackdriver) says above. And no .png file was created.

As for the "The get_cmap function was deprecated in Matplotlib 3.7 and will be removed in 3.11." problem, after i swaped 【line296 in finalize_hrtf_simulation.py (cmap=mpl.cm.get_cmap(name='hot_r'),)】with (cmap=mpl.colormaps['hot_r'],). the matplotlib problem works out.

python version = 3.9, pip list just like the requirements.txt.

**Comment by TheTrackdriver** [comment]
Yeah, that's the "finalize_hrtf_simulation.py" I'm trying to run. It does all the rest except the plotting.

I'm not a coder so I have to ask. What do you mean by "command"? What is the "environment" and how do I export it? Or doesn't it matter now because it's not a part of "mesh2hrtf.

**Comment by f-brinkmann** [comment]
It, might still be an issue with Mesh2HRTF. The error above says the problem is in finalize_hrtf_simulation.py, line 296 - you can check there. If that line does not use a part of the Mesh2HRTF API (https://mesh2hrtf.readthedocs.io/en/latest/mesh2hrtf.html), we undofrtunately can't support it here.

An `environment` usually referrs to specific versions of Python packages that you are using. To reprodice this I would at least need the version of Mesh2HRTF and Matplotlib, which you already listed above. So might work without.

**Comment by keithyipkw** [comment]
Changing `unit=None` to `unit='auto'` should fix the problem.

**Comment by f-brinkmann** [comment]
Thanks for figuring out and reporting. This is clearly a thing that should be update in mesh2hrtf_tools. I'll close the issue here but renamed it so it might be easier to find in case others run into the same problem...
